bridgev2/portal.go

```diff
@@ -3244,3 +3247,66 @@ func (portal *Portal) SetRelay(ctx context.Context, relay *UserLogin) error {
 	}
 	return nil
 }
+
+func handleMatrixMembership(
+	portal *Portal,
+	ctx context.Context,
+	sender *UserLogin,
+	origSender *OrigSender,
+	evt *event.Event) {
```

Comment:
The other handlers aren't methods because struct methods can't have their own type parameters, but this one doesn't have any type parameters

```suggestion
func (portal *Portal) handleMatrixMembership(
	ctx context.Context,
	sender *UserLogin,
	origSender *OrigSender,
	evt *event.Event,
) {
```